### PR TITLE
RUN-16860-added-info-for-editing-and-viewing

### PR DESCRIPTION
**Pull request by JamieWeider72**

### CODE REVIEW

**Review by jasonnovichRunAI — CHANGES_REQUESTED** [review]
@JamieWeider72 in the section with "Step 1: UI configuration". in step 3 there are only 2 choices, but you added a 3rd and did update the procedure.

File: docs/admin/runai-setup/authentication/sso.md [hunk 1]
```diff
@@ -12,33 +12,37 @@ Run:ai supports SSO using the [SAML 2.0](https://en.wikipedia.org/wiki/Security_
...
 
 * **Discovery URL**&mdash;the OpenID server where the content discovery information is published.
 * **ClientID**&mdash;the ID used to identify the client with the Authorization Server.
 * **Client Secret**&mdash;a secret password that only the Client and Authorization Server know.
 
 
-## OpenShift Prerequisites
+### OpenShift V4
 
 Before using OpenShift, first define OAuthClient to control various aspects of the OAuth flow, such as redirect URIs and authentication methods to ensure secure and approprpriate access to resources.
```
Comment (gal-revach) [inline docs/admin/runai-setup/authentication/sso.md:32]:
```suggestion
Before using OpenShift, first define OAuthClient. The OAuth client interacts with OpenShift’s OAuth server to authenticate users and request access tokens.
```

File: docs/admin/runai-setup/authentication/sso.md [hunk 2]
```diff
@@ -12,33 +12,37 @@ Run:ai supports SSO using the [SAML 2.0](https://en.wikipedia.org/wiki/Security_
...
 
 To define OAuthClient, follow these steps:
 
 1. Create a new ```OAuthClient``` Kubernetes object with the following content
```
Comment (gal-revach) [inline docs/admin/runai-setup/authentication/sso.md:36]:
```suggestion
1. Create a new ```OAuthClient``` Kubernetes object with the following content. 
Replace the `<runai_env_url>` with the URL of your Run:ai platform.
Replace `my-client` and `this-is-my-secret` with client name and secret you choose.
```

File: docs/admin/runai-setup/authentication/sso.md [hunk 3]
```diff
@@ -103,6 +107,14 @@ You can configure your IdP to map several IdP attributes:
...
+    	  
+    	Press `Edit` to view and edit the:
+    
+        * Discovery URL.
+        * Client ID.
+        * Client secret.
+
 	=== "OpenShift V4"
 
         1. In the `Discovery URL` field, enter the discovery URL .
```
Comment (gal-revach) [inline docs/admin/runai-setup/authentication/sso.md:120]:
```suggestion
        1. In the `Base URL` field, enter you OpenShift Base URL (https://api.<your-openshift-domain>:6443).
```

**Review by gal-revach — COMMENTED** [review]
@JamieWeider72 please see my comments

**Review by gal-revach — APPROVED** [review]

**Pull request merged**